src/components/work/Header/Header.tsx

```diff
@@ -39,16 +39,16 @@ const Header = () => {
   const [isOpen, toggleNavbar] = React.useState<boolean>(false);
 
   return (
-    <header className="fixed-top bg-white" id="bee-main-header">
+    <header className="fixed-top bg-white custom-header-height" id="bee-main-header">
```

Comment:
why can't we use the default header size with default font size provided?

Reply:
the issue here is that there is no default header size, it will shrink/grow to whatever the content is inside the nav.  Since we have a fixed-top nav this creates an issue with any hero/jumbotron.  It makes dealing iwth the header/padding easier if we set a fixed height to the nav so we know how much padding to add on hero/jumbotrons

Reply:
this page `front-v2.2/html/home/onepage-creative.html#homeSection` looks like it has a fixed header and works on mobile?

Reply:
this is because they set the padding from the hero content very tall, and both header/hero are with a white background.  then becomes more apparent when you have a blue hero and a white nav, you can see the space between here:

![screen shot 2019-02-22 at 12 26 46 pm](https://user-images.githubusercontent.com/13635403/53269235-36e12180-369d-11e9-9f4d-cd1d218a69d5.png)

Reply:
a few options:
1) set the padding manually on every page
2) the grey background is coming from the jumbotron, we can default the background color to white (this doesnt solve the underlying issue, basically just ignoring the fact that the padding/nav height arent equal